lib/eyes.js

```diff
@@ -0,0 +1,34 @@
+require('dotenv').config();
+
+const path = require('path');
+const uuid = require('uuid');
+const {Eyes} = require('eyes.images');
+
+const application = require(path.join(process.cwd(), 'package.json')).name;
+
+const theEyes = new Eyes();
+
+theEyes.setOs(process.platform);
+theEyes.setApiKey(process.env.EYES_API_KEY);
```

Comment:
If I get it right, this code will run when you require `eyes.js`, regardless you have api key or not, which might end with an error, in case you do not have an api key.

Reply:
I have already checked that manually. It allows to pass '', undefined, null value anything.

Reply:
The main question about this PR is that it changes the approach.
It's more like original `eyes.it`, but with less coupling and maybe more readable.

Reply:
It's not matcher anymore

Reply:
yes... we might consider maybe to move this to a different package, or change package name. I think that a different package will be cleaner. WDYT?

Reply:
ok, will move to package `with-eyes`

Reply:
I think that we will want to have at least an option for batching with the matcher (I didn't have time to think about it, yet). WDYT?